<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fabuRightWidget</class>
 <widget class="QWidget" name="fabuRightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改发布权限</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>角色名：</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="roleNameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>角色名</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>appName:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QComboBox" name="appComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="noOwnFabuLabel">
           <property name="text">
            <string>你暂为拥有的发布</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="allList"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>90</number>
         </property>
         <property name="bottomMargin">
          <number>90</number>
         </property>
         <item>
          <widget class="QToolButton" name="addToolButton">
           <property name="text">
            <string>增加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="delToolButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>你所拥有的发布</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="ownList"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="cacelButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sureButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
